tools/SetupFlow/DevHome.SetupFlow/Controls/SetupShell.xaml.cs

```diff
@@ -19,7 +19,11 @@ public sealed partial class SetupShell : UserControl
 {
     public string Title
     {
-        get => (string)GetValue(TitleProperty);
+        get
+        {
+            var title = (string)GetValue(TitleProperty);
+            return string.IsNullOrEmpty(title) ? Orchestrator.FlowTitle : title;

```

Comment:
That's a separate simplification I did, instead of having `UseOrchestratorTitle` and maintaining two `TextBlock`s, we can reduce it to a single `TextBlock`. I am hoping we can make bigger enhancements with the referenced issue.